src/main/java/net/reini/rabbitmq/cdi/MessageConverter.java

```diff
@@ -0,0 +1,25 @@
+package net.reini.rabbitmq.cdi;
+
+/**
+ * Transform a object in a array de bytes to send a message.
+ * 
+ * @author André Ignacio
+ */
+public interface MessageConverter {
+
+  /**
+   * Transform a object in a array de bytes to send a message.
+   *  
+   * @return Array of bytes
+   * @throws MessageConverterException If the conversion fails
+   */
+  byte[] toBytes(Object object) throws MessageConverterException;
```

Comment:
Here we would better use `byte[] encode(M message) throws MessageConverterException`